pkg/components/standalone_loader.go

```diff
@@ -54,7 +53,11 @@ func (s *StandaloneComponents) LoadComponents() ([]components_v1alpha1.Component
 	list := []components_v1alpha1.Component{}
 
 	for _, file := range files {
-		if !file.IsDir() && s.isYaml(file.Name()) {
+		if !file.IsDir() {
+			if !s.isYaml(file.Name()) {
+				log.Warnf("A non-YAML file %s was detected, it will not be loaded", file.Name())
```

Comment:
I would recommend making these logs more explicit

1. What kind of YAML file? Should specify if this is a component definition, etc
2. If possible, add a link to something. Could be this PR too, if there's a comment that's relevant.

Reply:
1. To add the kind of YAML file require read and parse the file first, I think that may cause unnecessary performance loss, I'm not sure that's necessary.
2. I will add a URL refer to this PR.

Reply:
For 1, is it not possible to guess the kind of file here? In this place, for example, you're reading components, so it may be possible to say that you found "A non-YAML component file"?

Reply:
Sorry for misunderstanding, it's ok to do so.